packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -329,7 +329,7 @@ class Frame extends React.Component {
     // and if the relatedTarget is falsey. The relatedTarget will be null when
     // dismissing the onscreen keyboard, else set to another element when
     // tapping elsewhere on the page
-    if (focusedByTouch && !relatedTarget) {
+    if (focusedByTouch && (relatedTarget === this.input)) {
```

Comment:
This inline comment ⬇️ may need to be updated now.

> Determined by if we have have interacted with the frame via onTouchStart and if the relatedTarget is falsey.

Reply:
Updated here https://github.com/cerner/terra-core/pull/2904/commits/09eb6be42fd6ff175b3d324f8d5ccbb9ca1583ec